### DISCUSSION TITLE
ValueError('faces.new(verts): face already exists

### DISCUSSION THREAD

**Discussion by macdroid53** [nortikin/sverchok#4518] [General]
I am getting this error when using Read FCStd node.

It happens on the second instance of an object in the FreeCAD file.

This is with sverchok 1.0.1. (On v1.1.0-alpha.1 it happens with both USB_micro_Touch and USB_micro_Power)

![image](https://user-images.githubusercontent.com/6831752/172859583-c53e3672-c931-4096-b396-d7612dc7c153.png)
[WIMAXIT_M1012_LCD.FCStd.zip](https://github.com/nortikin/sverchok/files/8870836/WIMAXIT_M1012_LCD.FCStd.zip)
[Assembly01-tst.blend.zip](https://github.com/nortikin/sverchok/files/8870911/Assembly01-tst.blend.zip)

I have different FreeCAD options for add shapes to a model, all result the same.

**Comment by zeffii** [comment]
issues discussed here #4522
